### PR TITLE
{tools}[foss/2018b] ITSTool 2.0.5 w/ Python 2.7.15 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node3154.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/3f0190517666b6b9586809178f1fe927 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2482.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/86c2a48f42400e67ebb837fe0214408d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2049.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6f20565b1d32b0189860cf72111dad98 for a full test report.

**Comment by boegel** [comment]
@hajgato Any idea why the one test report fails during `build` step?

```
failed to find headers for libxml2: update includes_dir
```

I tried re-installing the `libxml2` dep, but that didn't fix anything...

**Comment by hajgato** [comment]
@boegel I did not get at all that. I will check at other architectures.

**Comment by hajgato** [comment]
Test report by @hajgato
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
nic49 - Linux centos linux 7.5.1804, AMD Opteron(tm) Processor 6134, Python 2.7.5
See https://gist.github.com/d532045a401296ded7c24c501ebb1b7c for a full test report.

**Comment by hajgato** [comment]
Test report by @hajgato
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
nic400 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/89ca09a93ec016297e93ac027748374b for a full test report.

**Comment by hajgato** [comment]
Test report by @hajgato
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node001 - Linux RHEL 6.3, AMD Opteron(TM) Processor 6272, Python 2.6.6
See https://gist.github.com/dce677c11d2973c9eb059dcf067e047e for a full test report.

**Comment by boegel** [comment]
@hajgato Looks like the `setup.py` hardcodes location, so if you don't have `libxml2-devel` installed in the system where you're building `libxml2.Python`, this is doomed to fail... See https://github.com/ConradIrwin/libxml2/blob/master/python/setup.py#L72

And indeed, I don't have `libxml2-devel` installed on the system where the test report failed.

**Comment by hajgato** [comment]
@boegel It is not clear what are the consequences that libxml2 python bindings are build with EB provided iconv.h (libiconv) and `libxm2` was built with or without iconv.h, (or with a different version of iconv.h if libixml2 picked up it form system includes), but the python `setup.py` explicitly requires it, meanwhile it is only optional in the libxml2 configure script. In other words `libxml2` can be build without iconv.h (libiconv), but I suppose iconv.h is picked up from `libc-headers` in most of the cases and it can be different version what we provide in the libxml2.Python easyconfig.

**Comment by boegel** [comment]
@hajgato Please take a look at fixing the conflict at your earliest convenience (no rush)

**Comment by hajgato** [comment]
@boegel Done.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/4ca33bdabeb643ba124a9a3c9f83de60 for a full test report.

**Comment by migueldiascosta** [comment]
@hajgato, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

**Comment by migueldiascosta** [comment]
@hajgato thanks for the quick update - I think you'll also need to use https in `homepage`, for both easyconfigs

File: easybuild/easyconfigs/l/libxml2.Python/libxml2.Python-2.9.8-foss-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonPackage'
+
+name = 'libxml2.Python'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libxml2.Python/libxml2.Python-2.9.8-foss-2018b-Python-2.7.15.eb]: @hajgato Do you mind changing this to `libxml2-python`?

Cfr. my naming used in #7261 (but your approach using `PythonPackage` and using `libxml2` as a dep is better/cleaner)

Comment (hajgato) [reply easybuild/easyconfigs/l/libxml2.Python/libxml2.Python-2.9.8-foss-2018b-Python-2.7.15.eb]: @boegel Done!

**Comment by hajgato** [comment]
Is somebody up to merge this PR?

**Comment by boegel** [comment]
@hajgato I was mainly waiting for #7137 to get merged first, so I could do the cleanup done in https://github.com/hajgato/easybuild-easyconfigs/pull/37, should now be good to go...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/86200b166a730d273307e879064b9352 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3136.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/189f6f4f72c8712c6340f2f53350e72b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2420.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a5e53dfc36f1e8cedc8444d4673b20cf for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**